web/src/popover_menus.js

```diff
@@ -70,6 +76,26 @@ export function get_topic_menu_popover() {
     return popover_instances.topics_menu;
 }
 
+export function is_time_selected_for_schedule() {
+    return selected_send_later_time !== undefined;
+}
+
+export function get_selected_send_later_time() {
+    if (selected_send_later_time) {
+        // Add `at` between date and time which feels nicer to read.
+        return format(new Date(selected_send_later_time), "MMM d yyyy 'at' h:mm a");
+    }
+    return selected_send_later_time;
+}
```

Comment:
I don't understand this function -- this second case is just `return undefined` spelled differently right? I think it would be nicer written as:
```
if (!selected_send_later_time) {
    return undefined;
}
```

Reply:
Done.